decoder/vaapidecoder_h264.cpp

```diff
@@ -1520,15 +1521,89 @@ YamiStatus VaapiDecoderH264::decodeCurrent()
     return status;
 }
 
+uint32_t calcMaxDecFrameBufferingNum(const SharedPtr<SPS> sps)
```

Comment:
This code will call SharedPtr's copy constructor. const SharedPtr<SPS>& sps will not.